pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/tg_owt.nix

```diff
@@ -54,36 +56,50 @@ stdenv.mkDerivation {
     })
   ];
 
+  postPatch = lib.optionalString stdenv.hostPlatform.isLinux ''
+    substituteInPlace src/modules/desktop_capture/linux/wayland/egl_dmabuf.cc \
+      --replace '"libEGL.so.1"' '"${libGL}/lib/libEGL.so.1"' \
```

Comment:
```suggestion
      --replace '"libEGL.so.1"' '"${lib.getLib libGL}/lib/libEGL.so.1"' \
```
Same for the rest.

Reply:
Are you sure? libglvnd seem to have no lib output.

Reply:
`lib.getLib` always selects the correct _lib_ output (that is falling back to `out` if `lib` is not there)